packages/web3-core/types/index.d.ts

```diff
@@ -242,6 +242,7 @@ export interface Log {
     transactionHash: string;
     blockHash: string;
     blockNumber: number;
+    removed: boolean;
```

Comment:
@samlaf Thanks for your contribution, 
 Could you update [changelog](https://github.com/samlaf/web3.js/blob/patch-1/CHANGELOG.md) and mention this under 1.7.3

Reply:
Done. Please make sure I did it properly.